json-rpc/src/stream_rpc/subscription_types.rs

```diff
@@ -325,15 +325,15 @@ mod tests {
         // These should have no delay
         let expected =
             serde_json::json!({"jsonrpc": "2.0", "id": 1010101, "result":{"value": 0}}).to_string();
-        let next = timeout(10, receiver.recv(), "message 0")
+        let next = timeout(1_000, receiver.recv(), "message 0")
```

Comment:
what's the reason why we're even using timeouts in these tests? if we don't ever expect these to fail

Reply:
so that they fail fast in the event they are failing